.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ on:
   pull_request:
     branches: [main-application]
     paths:
-      - '.github/workflows/test.yml'
+      - '.github/workflows/**'
```

Comment:
Can we keep to `test.yml` we don't want to run this pipeline for IaC deployment.

Reply:
changed